_includes/dap.html

```diff
@@ -0,0 +1,3 @@
+<!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
+    <script async type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&subagency=18F" id="_fed_an_ua_tag"></script>
```

Comment:
I'd remove this indentation and the blank next line too

Reply:
If I remove the indentation, the markup ends up like this:
<img width="1307" alt="Screen Shot 2020-08-12 at 1 54 42 PM" src="https://user-images.githubusercontent.com/1053255/90050037-a7ba3600-dca3-11ea-9d93-eabfbf6ac220.png">
But no problem - I'll take it out.

Ironically, even after removing the extra newline in the partial, the source still has a newline between the script and `</body>`.

Reply:
Thx for the response!

Reply:
I feel like we should be optimizing for readability in each template, not the final HTML. If we wanted to optimize for that, maybe we should just have the tracking tag inline instead of being a partial?

Reply:
I agree with you - final HTML readability is not a necessary goal.  But I thought I'd test the waters.